chain/network/src/stats/metrics.rs

```diff
@@ -83,26 +81,26 @@ pub static REQUEST_COUNT_BY_TYPE_TOTAL: Lazy<IntCounterVec> = Lazy::new(|| {
 });
 
 // Routing table metrics
-pub static ROUTING_TABLE_RECALCULATIONS: Lazy<IntCounter> = Lazy::new(|| {
+pub(crate) static ROUTING_TABLE_RECALCULATIONS: Lazy<IntCounter> = Lazy::new(|| {
```

Comment:
Can you highlight this change in the description of the PR please?

Reply:
done